tasmota/settings.h

```diff
@@ -146,8 +146,8 @@ typedef union {                            // Restricted by MISRA-C Rule 18.4 bu
     uint32_t zb_received_as_subtopic : 1;  // bit 4 (v9.2.0.3)   - SetOption118 - (Zigbee) Move ZbReceived from JSON message and into the subtopic replacing "SENSOR" default
     uint32_t zb_omit_json_addr : 1;        // bit 5 (v9.2.0.3)   - SetOption119 - (Zigbee) Remove the device addr from json payload, can be used with zb_topic_fname where the addr is already known from the topic
     uint32_t zb_topic_endpoint : 1;        // bit 6 (v9.2.0.4)   - SetOption120 - (Zigbee) Append endpoint number to topic if device dependent (use with SetOption89)
-    uint32_t spare07 : 1;                  // bit 7
-    uint32_t spare08 : 1;                  // bit 8
+    uint32_t mqtt_state_retain : 1;        // bit 7 (development)- CMND_STATERETAIN
```

Comment:
Please follow the same format is above and take the current version

Reply:
is current version 9.3.0.1 ok to write here?

Reply:
Yes

Reply:
Ok, I also found it in tasmota_version.h,  thanx.

https://github.com/arendst/Tasmota/blob/73e2e4ff9ca777d9dda5031a95cf0f5992323ebf/tasmota/tasmota_version.h#L23